tests/end-to-end/save_to_h5.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# NVIDIA CORPORATION and its licensors retain all intellectual property
+# and proprietary rights in and to this software, related documentation
+# and any modifications thereto.  Any use, reproduction, disclosure or
+# distribution of this software and related documentation without an express
+# license agreement from NVIDIA CORPORATION is strictly prohibited.
+#
+echo ""
+echo "Read clean and noisy data in these intervals and save them in .h5 format..."
+echo ""
+# Training data
+python $root_dir/bw2h5.py \
+    --noisybw $data_dir/HSC.5M.chr123.10mb.coverage.bw \
+    --intervals $out_dir/result.training_intervals.bed \
+    --batch_size 4 \
+    --prefix $out_dir/train_data \
+    --cleanbw $data_dir/HSC.80M.chr123.10mb.coverage.bw \
+    --cleanpeakbw $out_dir/clean.peaks.bw \
+    --nonzero
+# Validation data
+python $root_dir/bw2h5.py \
+    --noisybw $data_dir/HSC.5M.chr123.10mb.coverage.bw \
+    --intervals $out_dir/result.val_intervals.bed \
+    --batch_size 64 \
+    --prefix $out_dir/val_data \
+    --cleanbw $data_dir/HSC.80M.chr123.10mb.coverage.bw \
+    --cleanpeakbw $out_dir/clean.peaks.bw
+# No label
+python $root_dir/bw2h5.py \
+    --noisybw $data_dir/HSC.5M.chr123.10mb.coverage.bw \
+    --intervals $out_dir/result.holdout_intervals.bed \
+    --batch_size 64 \
+    --prefix $out_dir/no_label \
+    --nolabel
+
+echo ""
+echo "Verifying output against expected results"
+python $test_dir/verify_diff.py --result_path $out_dir/train_data.h5 \
+	              --expected_path $expected_results_dir/train_data.h5 \
+		      --format "h5"
+if [ $? != 0 ]
```

Comment:
let's move this block into a function since it's used everywhere. and the function can take in as an argument a custom informative message to print on exit, e.g. "Failed to match train h5" or "Failed to match eval h5" instead of a more generic "Abort test"

Reply:
we can use the abstracted function across all tests

Reply:
custom messages are printed in python script. So, we don't need specific error messages here.

Reply:
Done. verify_diff already prints custom messages. Added a utils.sh script that has a function check_status that can print a custom message too.

Reply:
Ah I meant something a little different by the custom messages. verify_diff prints the diff failure between the file names, but sometimes file names get too long and hard to parse. So I was thinking of something like "save to h5 test failed", so in the logs it's easier to follow. we can update it in another PR though. can you file an issue and then resolve this conversation?

Reply:
done issue #116